Sources/Valet/SecureEnclave.swift

```diff
@@ -63,23 +66,30 @@ public final class SecureEnclave: Sendable {
     ///   - options: A base query used to scope the calls in the keychain.
     /// - Returns: The data currently stored in the keychain for the provided key.
     /// - Throws: An error of type `KeychainError`.
-    internal static func object(forKey key: String, withPrompt userPrompt: String, options: [String : AnyHashable]) throws -> Data {
+    internal static func object(forKey key: String, withPrompt userPrompt: String, options: [String : AnyHashable]) throws(KeychainError) -> Data {
         var secItemQuery = options
+#if !os(tvOS) && !os(watchOS) && canImport(LocalAuthentication)
         if !userPrompt.isEmpty {
-            secItemQuery[kSecUseOperationPrompt as String] = userPrompt
+            let context = LAContext()
+            context.localizedReason = userPrompt
```

Comment:
sigh. this API doesn't exist on either tvOS or watchOS. the methods that call into this method no longer expose the `userPrompt`